.gitattributes

```diff
@@ -0,0 +1,16 @@
+# Explicitly declare text files we want to always be normalized and converted on checkout.
+*.py text diff=python
```

Comment:
Interesting tidbit on what specifying the `diff` does: 

https://tekin.co.uk/2020/10/better-git-diff-output-for-ruby-python-elixir-and-more

Reply:
This is interesting, I actually never really look at the enclosing context, only line number :). But this is good to know!

Reply:
Yeah, same! Interesting little tidbit though!

Reply:
Kind of silly how you have to tell git that `*.rb diff=ruby`, `*.py diff=python`, but 🤷 ?